src/Countries/Pakistan.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+
+class Pakistan extends Country
+{
+    public function countryCode(): string
+    {
+        return 'pk';
+    }
+
+    protected function allHolidays(int $year): array
+    {
+        return array_merge([
+            'Kashmir Solidarity Day' => '02-05',
```

Comment:
You can also add translations: 
https://github.com/spatie/holidays/tree/main/lang

Reply:
Added